src/ashell/file-utils.c

```diff
@@ -78,3 +78,62 @@ int fs_close_alloc(fs_file_t *fp)
     zjs_free(fp);
     return res;
 }
+
+bool fs_valid_filename_size(char *filename)
+{
+    if (filename == NULL) {
+        printf("No filename given\n");
+        return false;
+    }
+
+    char *ptr1 = filename;
+    char *ptr2 = NULL;
+    ssize_t namelen;
+    ssize_t extlen;
+    ssize_t size = strlen(filename);
+
+    if (fs_exist(filename)) {
+        ptr1 = strchr(ptr1, '.');
+
+
+        if (ptr1 != NULL) {
+            // Check there aren't multiple periods
+            ptr2 = strchr(ptr1 + 1, '.');
+
+            if (ptr2 != NULL) {
+                printf("Invalid file extention format\n");
+                return false;
+            }
+
+            namelen = ptr1 - filename;
+            extlen = size - namelen - 1;
+        }
+        else
+        {
+            // Filename has no extension
+            namelen = strlen(filename);
+            extlen = 0;
+        }
+    }
+    else {
+        // Don't let the cfg be set to a file that doesn't exist
+        printf("File passed to cfg doesn't exist\n");
+        return false;
+    }
+
+    if (namelen == 0) {
+        printf("Filename length is zero\n");
+        return false;
+    }
+    else if (namelen > 8) {
+        printf("Filename is longer than 8 characters\n");
+        return false;
+    }
+
+    if (extlen > 3) {
+        printf("File extention is longer than 3 characters\n");
```

Comment:
Yeah looks like I got started thinking I needed to verify it was a valid file format, then realized all I needed to do was check if the file existed.  I'll clean this up